scripts/binary_size_check.sh

```diff
@@ -13,16 +13,16 @@ fi
 MAX_BINARY_MB=64
 MAX_BINARY_SIZE=$((MAX_BINARY_MB * 1024 * 1024))
 BIN_SUFFIX="-${ARCH}"
-if [ ${ARCH} = amd64 ]; then
+if [ "${ARCH}" = amd64 ]; then
     BIN_SUFFIX=""
-elif [ ${ARCH} = arm ]; then
+elif [ "${ARCH}" = arm ]; then
     BIN_SUFFIX="-armhf"
 fi
 
 CMD_NAME="dist/artifacts/k3s${BIN_SUFFIX}"
 SIZE=$(stat -c '%s' ${CMD_NAME})
 
-if [ ${SIZE} -gt ${MAX_BINARY_SIZE} ]; then
+if [ "${SIZE}" -gt |${MAX_BINARY_SIZE}| ]; then
```

Comment:
What is the intent of the pipes surrounding the variable here?

Reply:
This is my mistake, is must be double quotes.